app/responders/core_responder.rb

```diff
@@ -0,0 +1,18 @@
+module EventResponders
+
+    class CourseResponder < ApplicationResponder
```

Comment:
class names should be plural here, e.g. `CoursesResponder`

Reply:
Fixed